test/cli-commands/utils/spawn-child-process.js

```diff
@@ -42,17 +42,20 @@ function runCmdHandler(cmd, outputCondition, additionalCommand, secondAdditional
 		});
 
 		process.stderr.on('data', function (data) {
-			const err = data.toString('utf-8');
-			if (!err.includes('EADDRINUSE')) {
-				return resolve(err);
-			}
-
-			const errResponse = {
-				portInUse: true
-			};
-
-			resolve(errResponse);
-		});
+            const err = data.toString('utf-8');
+            if (!err.includes('EADDRINUSE')) {
```

Comment:
It seems to me that this is getting spaggettier? What will you do the next time when you need to catch an error that has error message of 'Something something XYZ'. Adding ifs to infinity is not a good solution, maybe you can find a better one where the expected string is somehow specified by the expector?